src/Knp/Component/Pager/Exception/PageLimitInvalidException.php

```diff
@@ -3,21 +3,29 @@
 namespace Knp\Component\Pager\Exception;
 
 use OutOfRangeException;
-use Throwable;
 
 final class PageLimitInvalidException extends OutOfRangeException
 {
-    private int $limit;
+    private ?int $limit = null;
```

Comment:
So, in the end, what's the purpose of keeping this property private, when it can be read and written in a public way?

Reply:
Afaik the use of public properties is not really common in Symfony code (https://www.reddit.com/r/PHP/comments/kkrzks/getterssetters_vs_public_properties/).

Reply:
It looks like none of the reasons for using getter/setter apply to this case.
Anyway, what's the purpose of exposing the value of the limit outside the exception class? I think that the message should be enough. So, if we want to keep it private, we should remove the getter.

Reply:
Well I just learned from other examples, to name a few:

https://github.com/KnpLabs/knp-components/blob/master/src/Knp/Component/Pager/Exception/PageNumberOutOfRangeException.php
https://github.com/symfony/symfony/blob/6.2/src/Symfony/Component/HttpKernel/Exception/HttpException.php

If it's just about the message, the property could be removed entirely.